src/measure_theory/l1_space.lean

```diff
@@ -395,6 +395,12 @@ begin
   exact integrable.smul _
 end
 
+lemma integrable.const_mul {f : α → ℝ} (h : integrable f μ) (c : ℝ) : integrable (λ x, c*f x) μ :=
+(integrable.smul c h : _)
+
+lemma integrable.mul_const {f : α → ℝ} (h : integrable f μ) (c : ℝ) : integrable (λ x, f x * c) μ :=
+by simp_rw [mul_comm, h.const_mul _]
```

Comment:
the spacing is strange : you have `c*f x` and `f x * c`. I tend to prefer the latter `f x * c`, but I value coherence above all!